docs/auto-trace-span-extraction.rst

```diff
@@ -0,0 +1,27 @@
+Automatic Trace/Span ID Extraction
+==================================
+
+.. note::
+    All `LogEntry fields`_ populated :ref:`manually<Manual-Metadata>` will override those populated via methods referred to in this
+    section.
+
+The Google Cloud Logging library can automatically populate `LogEntry fields`_
+`trace`, `span_id`, and `trace_sampled` via OpenTelemetry integration, or extracting header information from an HTTP request.
+
+OpenTelemetry Integration
+-------------------------
+
+If you have the OpenTelemetry SDK package installed and are logging from within an active OpenTelemetry span, that log entry will automatically
```

Comment:
nit the SDK is not technically necessary here. Does this suggestion make sense to you? I think it's a little more precise
```suggestion
If you are using OpenTelemetry and there is an active span in the OpenTelemetry Context, that log entry will automatically
```